docs/docs-content/enterprise-version/install-palette/install-on-kubernetes/uninstall.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: "Uninstallation"
+title: "Uninstall Palette"
+description: "Learn how to uninstall a Palette installation from your cluster using Helm charts."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["self-hosted", "enterprise"]
+keywords: ["self-hosted", "enterprise"]
+---
+
+To uninstall Palette from your cluster, you need to uninstall the following components: Reach system, Palette management
+plane, and Cert Manager. This uninstall process applies to both connected and airgap instances of self-hosted Palette.
+
+## Prerequisite
+
+- An active self-hosted Palette instance.
+
+- Access to the Kubernetes cluster where Palette is deployed.
+
+- Permissions to perform delete actions on all relevant cluster resources.
+
+- Helm is installed and available on your host machine.
+
+- kubectl is installed and available on your host machine.
+
+## Uninstall Palette
+
+1. Ensure you are using the appropriate context where Palette is deployed.
+
+   ```shell
+   kubectl config current-context
+   ```
+
+2. Issue the following command to start uninstalling Reach. This will remove all resources related to Reach that are
+   managed by Helm. However, some resources created by Helm hooks are not managed by helm and will require additional
+   manual intervention to remove.
+
+   ```shell
+   helm uninstall reach-system
+   ```
+
+3. Issue the following command to remove the remaining Reach system resources.
+
+   ```shell
+   kubectl get all,configmap,secret,serviceaccount,mutatingwebhookconfiguration,validatingwebhookconfiguration,clusterpodpreset --output=jsonpath='{range .items[?(@.metadata.annotations.helm\.sh/hook)]}{.kind}/{.metadata.name}{"\n"}{end}' --namespace reach-system | grep -v "ReplicaSet/" | xargs -r kubectl delete --namespace reach-system && kubectl delete ns reach-system && kubectl delete crd podpresets.reach.spectrocloud.com clusterpodpresets.reach.spectrocloud.com
+   ```
```

Comment:
For removing remnant reach-system resources, I want us to document explicitly which resources need to be removed:
```
kubectl delete ns reach-system
kubectl delete crd podpresets.reach.spectrocloud.com clusterpodpresets.reach.spectrocloud.com
kubectl delete mutatingwebhookconfiguration reach-mutating-webhook-configuration
kubectl delete validatingwebhookconfiguration reach-validating-webhook-configuration
kubectl delete clusterrolebinding reach-manager-rolebinding
kubectl delete clusterrolebinding reach-proxy-rolebinding
kubectl delete clusterrole reach-manager-role
kubectl delete clusterrole reach-metrics-reader
kubectl delete clusterrole reach-proxy-role
```